src/utils.js

```diff
@@ -13,6 +13,20 @@ import { PACKAGE_TYPES, LOCAL_PROTOCOLS } from 'const';
 
 const SOURCE_MAP_RE = new RegExp(/\/\/[#@]\s(source(?:Mapping)?URL)=\s*(\S+)/);
 
+export function errorParamsToUnsupportedVersionRange(errorParams) {
+  const { min_manifest_version, max_manifest_version } = errorParams || {};
+  if (min_manifest_version != null || max_manifest_version != null) {
+    return [
+      min_manifest_version ? `< ${min_manifest_version}` : undefined,
+      max_manifest_version ? `> ${max_manifest_version}` : undefined,
+    ]
+      .filter((e) => e !== undefined)
+      .join(' or ');
+  }
+
+  return '';
```

Comment:
This should probably return `null` even if it does not impact the condition (in one of the files above).

Reply:
My personal preference for this helper would be to keep the same return type in both cases, either when there is a range and when there isn't any.

At the moment the return type is always 'string' and so the caller can just interpolate the return value into a template string as is without bothering to convert the null into an empty string (`${null}` => 'null'`, on the contrary I want it to be an empty string when there is no range).

Reply:
:+1: